src/index.js

```diff
@@ -72,11 +71,11 @@ module.exports = function emberCliUpdate(options) {
 
     let shouldRunModulesCodemod = semver.lt(startVersion, modulesCodemodVersion) && semver.gte(endVersion, modulesCodemodVersion);
     if (shouldRunModulesCodemod) {
-      return new Promise(resolve => {
-        let cp = fork(modulesCodemodPath);
-
-        cp.on('exit', resolve);
-      }).then(() => {
+      let opts = {
```

Comment:
Do we also have to set `preferlocal` here?

Reply:
No, it's default is `true`

Reply:
Although we could to be more explicit I suppose

Reply:
Nah, seems good if it’s being defaulted. I didn’t realize it was the default.